backend/src/routes/notes.ts

```diff
@@ -0,0 +1,23 @@
+import express, { NextFunction, Request, Response } from "express";
+import { Note } from "../models/notes";
+
+const router = express.Router();
+
+router.get("/notes/:id", async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    console.log("Getting...");
+    const id = req.params.id;
+    const note = await Note.findById(id);
+    if (note == null) {
+      throw new Error();
+    }
+    return res.status(200).json(note.answers);
```

Comment:
We need to send the question in this GET response as well. The frontend will not hardcode the questions but will rely on what comes from the backend

Reply:
@christen03 lets add this and then we should be good to merge this